docs/engage/front-end-ui/admin-section/_category_.json

```diff
@@ -0,0 +1,8 @@
+{
+    "label": "Engage Admin UI",
+    "position": 1,
```

Comment:
I think the admin section can go last in the navigation dosent have to be position 1